scripts/zones/Pashhow_Marshlands/mobs/BoWho_Warmonger.lua

```diff
@@ -3,16 +3,64 @@
 --   NM: Bo'Who Warmonger
 -----------------------------------
 require("scripts/globals/regimes")
+mixins = {require("scripts/mixins/job_special")}
+local ID = require("scripts/zones/Pashhow_Marshlands/IDs")
 -----------------------------------
 local entity = {}
 
+-- TODO: Implement better pathing systems for guards to follow master
+
+entity.onMobSpawn = function(mob)
+    -- Takes half damage from all attacks
+    mob:addMod(xi.mod.DMG,-5000)
+
+    -- May spawn in a party with two other Quadav
+    if math.random(1,2) == 1 then
```

Comment:
Been thinking about the helpers all these get and I am actually wondering if they are bugged in some way on retail, I see them so rarely. How do you feel about using
`if math.random(3) == 2 then` 
which would be a 1 in 3 chance instead of 50/50

We don't have any real data as to their spawn rates. or why they do or don't pop. I left the Antican one up for a few hrs one day just following without agroing it to see if maybe they needed some time afterword to pop, or if they spawn separately and then link up. Never saw them. Another time after maint they were just all 3 there when I arrived. Possibly we'll never really know without SE telling us.

The PR does not hinge on anything here changing.

Reply:
I'm more than happy to adjust that since I don't have a ton of experience with them personally.

The main thing I would really love to implement for the guards is to follow the same pathing logic BST pets do since they're really wonky right now with my current method. Just wasn't sure if there was a way to replicate that in lua. Would be nice since there is an identical NM for nearly every Beastmen type.

Reply:
I can't recall where I saw it but there was something that did pathing in its on roam to trail behind another mob. But I suspect an even better option is to copy what mob's pets do. Kind of make these a pseudo pet that doesn't always pop and doesn't get forced to agro together (usually tough to do but the guards can be separated - at least for the ones that don't detect by sound anyway). Maybe we could even reuse that system but make a few mods to just exclude those things via some new mobmods.

I'd have to do some digging to look at that myself but it seems a possibility. We can do said rigging after the fact, whichever way it has to be done. It'll likely set the standard for how to make mobs follow a leader, so we'll need to brainstorm - lets make a discussion topic.

Reply:
Well I think #2605 might be the solution to this problem!